ui/desktop/README.md

```diff
@@ -41,6 +41,7 @@ List of available project commands.  `yarn run <command-name>`
 | ------- | ----------- |
 | build:development | Builds the UI in development mode. |
 | build | Builds the UI for production. |
+| build:desktop:debian | Builds debian based linux artifacts on MacOS. |
```

Comment:
neat!

Reply:
@gsusmi Is this command _specifically_ for building on macOS?  If so, the command itself should probably reflect that.  If not, the description should be changed.

Reply:
It is specially for building linux artifacts on macOS. Will update.

Reply:
Updated script name to `build:desktop:debianOnMacOS`.